tests/cli/core/products/services/test_product_service.py

```
@@ -33,7 +34,7 @@ def test_create(mocker, service_context, mpt_product_data, product_data_from_dic
     api_update_mock = mocker.patch.object(service_context.api, "update")
     write_id_mock = mocker.patch.object(service_context.file_manager, "write_ids")
     stats_spy = mocker.spy(service_context.stats, "add_synced")
-    service = ProductService(service_context)
+    service = ProductService(service_context)  # noqa: WPS204
```

Ruff (0.15.1)

[warning] Unused `noqa` directive (unknown: `WPS204`)

Remove unused `noqa` directive

(RUF100)